tests_v2/test_cli_new_workspace_background_metadata.py

```
@@ -0,0 +1,191 @@
+            )
+
+            _must(
+                c.current_workspace() == baseline_workspace,
+                "background metadata load should not switch selected workspace",
+            )
+    finally:
+        if created_workspace:
+            try:
+                _run_cli(cli, ["close-workspace", "--workspace", created_workspace])
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)